### ISSUE TITLE
Why is the effect of using pre-training worse than not using it when training on ISIC2018?（isic2018训练使用预训练的效果比不使用更差了为什么）

### ISSUE THREAD

**Issue by jiangyanbin51-netizen** [JCruan519/VM-UNet#113]
After using pre-training, I found that the training became unstable, and the loss was higher and more unstable than without pre-training. May I ask what's going on here?（使用了预训练之后发现训练变得不稳定，loss比不使用预训练的时候更高更不稳定，麻烦问下这是什么情况？）
